<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Company</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="4" column="0" colspan="3">
     <widget class="QPushButton" name="btnDeleteProfit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>&amp;Delete Selected Profit Company</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QListWidget" name="listWidgetProfit">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <widget class="QListWidget" name="listWidgetNonProfit">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>List of companies for non profit</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QLabel" name="lblList">
      <property name="text">
       <string>List of companies for profit</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="boxData">
      <property name="title">
       <string>Enter company data</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="lblDate">
         <property name="text">
          <string>Date</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDateEdit" name="datDate"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblName">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="txtName">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblEmployees">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Employees</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblCharitable">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Charitable?</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="chkCharitable">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spnEmployees">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>99999999</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="btnAdd">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Add company</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="boxCompanyType">
      <property name="title">
       <string>Select a company type</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="radProfit">
         <property name="text">
          <string>For Profit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radNotProfit">
         <property name="text">
          <string>Not for Profit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="QPushButton" name="btnDeleteNonProfit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>&amp;Delete Selected Non Profit Company</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
